src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -79,6 +79,11 @@
     private String prefix;
     @Deprecated private transient String region, credentialsId;
 
+    /**
+     * Flag to use accelerated endpoint.
+     */
+    private Boolean acceleratedEndpoint;
```

Comment:
`boolean` not `Boolean`.

Where are you getting this name? The docs do not mention anything about an “accelerated endpoint”. They refer to “transfer acceleration” and the APIs refer to “accelerate” or “accelerate mode”. We should use consistent terminology, both in the code and UI.